NAMESPACE

```diff
@@ -140,10 +136,7 @@ import(maftools)
 import(metaviz)
 import(parallel)
 import(plotly)
-import(randomForest)
-import(reshape2)
 import(rtracklayer)
-import(stats)
```

Comment:
From the documentation overhaul that I am working on, I think this package is still being used by at least one existing GAMBLR function (`FtestCNV`). I could be wrong, what are your thoughts?

Reply:
Thanks for noticing, stats should be dropped from that function too. I'll push the fix!

Reply:
this is now done ✅